recipe/meta.yaml

```diff
@@ -48,6 +48,25 @@ outputs:
         - if not exist %LIBRARY_BIN%\zlib.dll exit 1  # [win]
         - if not exist %PREFIX%\zlib.dll exit 1  # [win]
 
+  - name: libzlib-wapi
+    script: bld_winapi.bat   # [win]
+    build:
+      skip: True   # [not win]
+      run_exports:
+        - {{ pin_subpackage('libzlib-wapi', max_pin='x.x') }}
+    requirements:
+      build:
+        - {{ compiler('c') }}
+      host:
+      run:
+      run_constrained:
+        - zlib {{ version }} *_{{ build_num }}
+    files:
+      - Library/bin/zlibwapi.dll   # [win]
```

Comment:
You have a script, so listing files shouldn't be relevant and might actually break in future conda-build releases.

Reply:
Hmm, okay, I can probably remove that. I was following the pattern for `libzlib` package in the same file.

Reply:
`libzlib` doesn't have a script.

Reply:
I'm sorry I don't get it. `libzlib` does have a script, it's `bld.bat`, that installs libraries in the lib prefix and copies a few files. It has a default name so it's not required to be specified in the recipe.
I removed this "files:" section.